scss/bitstyles/layout/box-alignment/_.scss

```diff
@@ -13,3 +13,7 @@
 .l-items-end {
   align-items: flex-end;
 }
+
+.l-items-stretch {
+  align-items: stretch;
+}
```

Comment:
Why has only the first utility class in this file the `#{bitstyles-namespace}`-prefix?

Are there other `align-items:` values that might warrant such a helper? Most others seem rather uncommon, but maybe `baseline`? 🤷

Reply:
Arg, yup I always forget about that, thanks! (it’s a feature I’ve never needed in practice actually) I’ll add 👍